tests/__snapshots__/index-spec.js.snap

```diff
@@ -0,0 +1,4428 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Plugin must contain configs and rules properties 1`] = `
+Object {
+  "base": Object {
+    "env": Object {
+      "es6": true,
+      "node": true,
+    },
+    "parser": "/home/rusinov/yola/eslint-plugin-yola/node_modules/babel-eslint/index.js",
```

Comment:
This snapshot fails for me:

```
 FAIL  tests/index-spec.js
  ✕ Plugin must contain configs and rules properties (107ms)

  ● Plugin must contain configs and rules properties

    expect(value).toMatchSnapshot()

    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

    @@ -2,11 +2,11 @@
        "base": Object {
          "env": Object {
            "es6": true,
            "node": true,
          },
    -     "parser": "/home/rusinov/yola/eslint-plugin-yola/node_modules/babel-eslint/index.js",
    +     "parser": "/Users/doug/Yola/eslint-plugin-yola/node_modules/babel-eslint/index.js",
```

On a whole, I tend to dislike tests that rely on snapshots because they're very brittle and break for nonsensical reasons like this.

Is it possible to get some coverage without a snapshot?

Reply:
Yes it is possible. I think the code is pretty unit-like atm. I would also leave snapshots but for shallow object structure tho. It will be snapshot  of immediate child properties of `plugin.configs` and `plugin.rules`.